mesh/fake-mesh-env.sh

```diff
@@ -0,0 +1,26 @@
+# To use any of the optional configuration options uncomment that line. If you no longer want to use an optional
```

Comment:
rename to env.fake-mesh.sh for consistency? Also updated README "Using with Fake MESH" section to instruct copying this file to env.sh